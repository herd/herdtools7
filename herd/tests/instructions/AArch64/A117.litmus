AArch64 A117
(* unconditional Branch instruction *)

 { 0:X0=1; }

P0;
  B foo;
  ADD W0, W0, #1;
  foo: NOP;

forall (0:X0=1)
