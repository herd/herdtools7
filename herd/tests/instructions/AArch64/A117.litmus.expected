Test A117 Required
States 1
0:X0=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=1)
Observation A117 Always 1 0
Hash=1e56e7a378d422667a5081fc63e22f1e

